web/crux-ui/tsconfig.json

```diff
@@ -30,15 +21,10 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "typeRoots": ["node_modules/yup"]
+    "typeRoots": ["node_modules/yup"],
+    "outDir": "./dist"
```

Comment:
we shouldn't really touch tsconfig, cause it's handled by next
i think especially 'outDir' could cause issues.

https://nextjs.org/learn/excel/typescript/create-tsconfig